src/NetSparkle.Tools.DSAHelper/NetSparkle.Tools.DSAHelper.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProductVersion>8.0.30703</ProductVersion>
-    <ProjectGuid>{B4D6FCB0-1217-4AFA-A2A5-4BA5482876C0}</ProjectGuid>
```

Comment:
I realize these changes might be coming from Visual Studio for macOS, but please don't commit/bring into this PR all the Guid removals, SLN changes, etc. (AKA things that aren't really things being handled/changed by this PR.)

Reply:
I think this is happening because I have to unload like 50% of the project to make it work. No idea why VS is doing this. Do you want me to try to revert this?

Reply:
It's VS being a pain, to be sure -- I saw similar things happening on macOS here. 

Please revert them -- I think the easiest thing to do on your end would probably be to do something like:

```bash
git reset HEAD~3 # go back 3 commits or however many you've done
git add _files_to_actually_commit_not_sln_csproj
git commit -m "Message"
git push -f
```

I'm OK losing the "history" that you have with the 3 separate commits. Not a huge deal at all.

Reply:
I think I can just copy the original sln over it to resolve and squash it.

Reply:
Yeah, that would work too.